d2l/mxnet.py

```diff
@@ -1,4 +1,9 @@
-# This file is generated automatically through:
+USE_MXNET = True
+USE_PYTORCH = False 
+USE_TENSORFLOW = False
+
+#################   WARNING   ################
```

Comment:
Using https://github.com/d2l-ai/d2l-book/commit/ca1276c7453c322fa69b965e33f6c2779e81f1de#diff-2911d8692efa3ddf62c3d0e50a8e1ac45393b929073b7a17e46309b8d07781a8, these `USE_XX` flags are not generated from `build lib`. Did I miss anything?